handler.php

```diff
@@ -7,7 +7,7 @@
 class handler {
 
     /** Associative array that maps building-ids to addresses */
-    private $buidings;
+    private $buildings;
```

Comment:
I first thought you were missing to rename the other occurrences of this variable, but it looks like it's not used at all anymore?

Reply:
Before, the constructor was dynamically creating the property with correct spelling. So while the declaration is not strictly necessary, it is considered a good practice to have it. This way, the visibility is properly set.

Reply:
Thanks for clarification!